client/js/components/fragments/Map.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import GoogleMapsLoader from 'google-maps';
+
+class Map extends Component {
+  constructor () {
+    super();
+    this.map;
```

Comment:
I'm not using getInitialState anywhere. All I'm saing that the constructor above does not do anything IMO and can be removed